rkt/pubkey/pubkey.go

```diff
@@ -227,6 +235,36 @@ func downloadKey(u *url.URL) (*os.File, error) {
 	return retTf, nil
 }
 
+func getClient(skipTLSCheck bool) *http.Client {
+	if !skipTLSCheck {
+		return http.DefaultClient
+	}
+	client := *http.DefaultClient
+	var tr *http.Transport
+	// default transport is hidden behind the RoundTripper
+	// interface, so we can't easily make a copy of it
+	if transport, ok := http.DefaultTransport.(*http.Transport); ok {
```

Comment:
In general - in version 1.4, 1.5 and 1.6, DefaultTransport is an instance of http.Transport hidden behind the RoundTripper interface, so for these versions we have `ok`. I added the `!ok` case, just because I could. :P But I guess I should just panic here and when an incompatible change is made in go, we can adapt. The comment in 250 is simply saying that the `ExpectContinueTimeout` field exist only in go 1.6 and later, so we don't set it even if go1.6 sets it to some value, because we want the code to build with older go too.

So in short - `!ok` should not happen, should I panic here?